subprojects/shipkit/src/test/groovy/org/shipkit/gradle/ShipkitGradlePluginIntegTest.groovy

```diff
@@ -1,10 +1,15 @@
 package org.shipkit.gradle
 
+import org.gradle.testkit.runner.BuildResult
 import testutil.GradleSpecification
 
 class ShipkitGradlePluginIntegTest extends GradleSpecification {
```

Comment:
I would keep them in a separate sourceset otherwise:

![screenshot from 2017-09-16 10-49-26](https://user-images.githubusercontent.com/148013/30510851-e08e78e0-9acc-11e7-8bf9-6eda1b45856e.png)

Anyway, by default, just the current version is tested from IDE. It can be tweaked in Idea test runner to have more version tested if anyone needs to do that